client/build/index.html

```diff
@@ -5,7 +5,9 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <script src="bundle.js"></script>
-  <title>Document</title>
+  <title>Space Oddity</title>
```

Comment:
These are going to clash with mine but only because we've done the same thing on different lines.

Reply:
Ahh yeh, no worries though thats an easy conflict fix